external-crates/move/crates/move-compiler/tests/move_check/typing/type_variable_join_single_pack.exp

```diff
@@ -7,27 +15,74 @@ error[E01002]: unexpected token
   │                   Unexpected 'copy'
   │                   Expected '{'
```

Comment:
Can we hand-tune the parsing here to skip to the struct body properly?

Reply:
Initially I wanted to limit this PR in scope and not handle struct parsing resilience just yet. I can see how the number of additional error messages would be reduced here making this PR cleaner so I added it (in a similar style as function parsing resilience).